### ISSUE TITLE
Empty debug log

### ISSUE THREAD

**Issue by JustinDrake** [OpenBazaar/OpenBazaar-Client#1274]
On Mac, clicking `View Debug Log` opens an empty file. Is that by design?

![screen shot 2016-04-06 at 10 45 49](https://cloud.githubusercontent.com/assets/731743/14312576/c00e98b2-fbe4-11e5-98d6-aa02fdffba6e.png)

**Comment by jjeffryes** [comment]
@hoffmabc @morebrownies can either of you confirm if this is still an issue?

**Comment by sgornick** [comment]
On Linux I get:

"There was an error and we are unable to open the server debug log at this time.

Error: ENOENT: no such file or directory, open '/opt/OpenBazaar-Client-1.1.3/../OpenBazaar-Server//debug.txt"

![openbazaar-debug-log](https://cloud.githubusercontent.com/assets/24884/14579196/22929f2a-0355-11e6-936c-64c7ea694aea.png)

OpenBazaar Client, v1.1.3

**Comment by jonathancross** [comment]
@jjeffryes  I am still seeing this on Mac OSX, but client is about 2 months old: `1.1.9`

**Comment by hoffmabc** [comment]
Yes I can confirm this is still a bug. I'll try to get a fix in for this.

**Comment by jonathancross** [comment]
On my system (OSX), the file being opened via status menu is:`OpenBazaar-Server/debug.txt`
But the actual debug log should be: `~/Library/Application Support/OpenBazaar/debug.log`
